tests/distributions/test_custom_distribution.py

```diff
@@ -0,0 +1,662 @@
+#   Copyright 2024 The PyMC Developers
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+import warnings
+
+import cloudpickle
+import numpy as np
+import numpy.random as npr
+import pytensor
+import pytensor.tensor as pt
+import pytest
+import scipy.stats as st
+
+from pytensor import scan
+
+from pymc import Bernoulli as pm_Bernoulli
```

Comment:
Why the alias? Should just be `from pymc.distributions import Bernoulli, Beta, ...`

Reply:
I just realised that after pushing the commit, that's fixed in e365f4c where the alias is only applied to those imports that would conflict